clusters/build-clusters/01_cluster/machine_config/m5d_machineconfig.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
```

Comment:
I'm not 100% sure on this part, but I think this should be something like worker-m5d4x

Reply:
Done.